dept44-api-models/src/main/java/se/sundsvall/dept44/api/models/paging/PagingMetaData.java

```diff
@@ -42,7 +43,7 @@ public class PagingMetaData {
 	@Schema(description = "The properties to sort by", example = "property", accessMode = READ_ONLY)
 	private List<String> sortBy;
 
-	@Schema(description = "The sort order direction", example = "ASC", enumAsRef = true)
+	@ArraySchema(schema = @Schema(description = "The sort order direction", example = "ASC", enumAsRef = true))
```

Comment:
Shouldn't `@ArraySchema` be placed on sortBy-variable instead?

('sortBy' is a List, 'sortDirection' is not a List)